src/test/scala/is/hail/variant/vsm/GroupBySuite.scala

```diff
@@ -0,0 +1,70 @@
+package is.hail.variant.vsm
+
+import is.hail.SparkSuite
+import is.hail.expr.{TFloat64, TInt64}
+import is.hail.io.annotators.IntervalList
+import org.testng.annotations.Test
+import is.hail.TestUtils._
+import is.hail.utils._
+
+
+class GroupBySuite extends SparkSuite {
+
+  @Test def testGroupVariantsBy() {
+    val vds = hc.importVCF("src/test/resources/sample.vcf").annotateVariantsExpr("va.foo = gs.filter(g => g.isDefined).map(g => g.gt).sum()")
+    val vds2 = vds.groupVariantsBy("va.foo", "gs.map(g => g.gt).max()", true)
+  }
+
```

Comment:
I think you should test handling of missing and some form of identity under trivial grouping on random vsm